gtfs2ntfs/src/main.rs

```diff
@@ -144,7 +144,30 @@ fn run(opt: Opt) -> Result<()> {
         None,
     )?;
 
-    transit_model::ntfs::write(&model, opt.output, opt.current_datetime)?;
+    match opt.output.extension() {
+        Some(ext) if ext == "zip" => {
+            let output_dir = tempfile::TempDir::new()?;
+            transit_model::ntfs::write(&model, output_dir.path(), opt.current_datetime)?;
+            let zip_file = std::fs::File::create(opt.output)?;
```

Comment:
Why you don't use `pub fn zip_to<P, R>(source_path: P, zip_file: R) -> crate::Result<()>` in utils ?

Reply:
Mostly because I didn't know its existence... :smile:

Reply:
But now that I know about it... it's a private method... and I'm wondering if this the job of a library like `transit_model` to expose such a function.

Reply:
Finally, I'm using `transit_model::ntfs::write_to_zip()` that I didn't know about but @patochectp mentioned to me in private.